ister_test.py

```diff
@@ -1321,20 +1321,6 @@ def add_bundles_good():
     commands_compare_helper(commands)
 
 
-@open_wrapper("good", "1\n")
-def get_current_format_good():
```

Comment:
I expect the test needs to be removed from the tests array in this commit too.

Reply:
It's already taken out ;)

Reply:
Ah, okay sorry I didn't see that X(.